pages/dkp/kommander/2.2/release-notes/index.md

```diff
@@ -2,10 +2,185 @@
 layout: layout.pug
 navigationTitle: Kommander 2.2 Release Notes
 title: Kommander 2.2 Release Notes
-menuWeight: 0
+menuWeight: 10
 excerpt: View release-specific information for Kommander 2.2
 enterprise: false
 beta: false
 ---
 
-This section includes the Release Notes for Kommander 2.2
+**D2iQ&reg; Kommander&reg; (DKP&reg;) version 2.2 was released on March 16, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+[Download](../../download/) and [install](../../install/) the latest version to get started.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download Kommander. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+Kommander 2.2 supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range. 
+
+### New features and capabilities
+
+#### DKP major version upgrade
```

Comment:
Take this whole section out. Major version upgrade was ONLY for 2.1. We won't support this anymore in 2.2 and have already removed the topics that tell people how to do this.